src/Modules/Settings.php

```diff
@@ -198,6 +220,68 @@ public function user_registration(): void {
 		<?php
 	}
 
+	/**
+	 * Toggle One Tap Login functionality.
+	 *
+	 * @return void
+	 */
+	public function one_tap_login(): void {
+		?>
+		<label style='display:block;margin-top:6px;'><input <?php $this->disabled( 'one_tap_login' ); ?>
+					type='checkbox'
+					name='wp_google_login_settings[one_tap_login]'
+					id="one-tap-login" <?php echo esc_attr( checked( $this->one_tap_login ) ); ?>
+					value='1'>
+			<?php esc_html_e( 'One Tap Login', 'login-with-google' ); ?>
+		</label>
+		<?php
+	}
+
+	/**
+	 * One tap login screens.
+	 *
+	 * It can be enabled only for wp-login.php OR sitewide.
+	 *
+	 * @return void
+	 */
+	public function one_tap_login_screens(): void {
+		$default = $this->one_tap_login_screen ?? '';
+		// phpcs:disable
```

Comment:
This will disable all the PHPCS rules. Only disable specific rules only.

Reply:
Done.

Reply:
I mean mentioned rules you want to ignore. If rules not mentioned then all rules will be ignored. Also enable ignored rules.